Sources/main.cpp

```diff
@@ -511,9 +569,17 @@ int main(int argc, char **argv) {
     UsageDie(argv[0]);
   }
 
-  switch (argv[1][0]) {
-  case 'g':
+  if (argv[1][0] == 'g')
```

Comment:
You can move the close loop before this line and leave the `switch ()` block untouched.